test-crawler/collector/collector.go

```diff
@@ -126,3 +129,29 @@ func ListGoFilesInFolder(root string, ignore []string) (system string, files []s
 
 	return "", nil, nil
 }
+
+// RegenerateName regenerates name adding underscore and usecase index
+// ex: name: ListenProofRequest -> LISTEN_PROOF_REQUEST_001
+func (f *FunctionAnnotation) RegenerateName() {
```

Comment:
I would rather have this function return the value of the new name, so the caller can set it.
We are mutating the structure right now (ie. overwriting the value of `f.Name`), but in the future, we will create a new structure, to keep the logic separate.